config/config.go

```diff
@@ -47,13 +49,23 @@ var (
 
 	// errEarlyReturn is returned to break out of a loop from a callback early
 	errEarlyReturn = errors.New("early escape")
+
+	LogLevels = enumnames.NewMap(map[logr.Level]string{
+		logr.LevelDebug: "DEBUG",
+		logr.LevelInfo:  "INFO",
+		logr.LevelWarn:  "WARN",
+		logr.LevelError: "ERROR",
+	})
 )
 
 // Config is the main configuration structure
 type Config struct {
 	// Debug Whether to enable debug logs
 	Debug bool `yaml:"debug,omitempty"`
 
+	// Log level: DEBUG, INFO, WARN, ERROR.  Default: INFO
+	LogLevel string `yaml:"log-level,omitempty"`
```

Comment:
I just made a couple changes to the library - you should now be able to just use `logr.Level` as I've made it into a string.

This way, you won't have to worry about converting anything.